modules/_labs/combobox/react/lib/Combobox.tsx

```diff
@@ -125,6 +129,7 @@ export default class Combobox extends React.Component<ComboboxProps, ComboboxSta
   static defaultProps = {
     clearButtonLabel: `Reset Search Input`,
     clearButtonVariant: IconButton.Variant.Plain,
+    useLegacyAriaAttributes: true,
```

Comment:
I have an open question with our accessibility team asking if we should support both ARIA 1.0 and 1.1 for combobox. There are issues with both. More context to the issue with the 1.1 spec: https://github.com/w3c/aria/wiki/Resolving-ARIA-1.1-Combobox-Issues

Pending the answer to the question, this might be a good time to add accessibility specifications to the combobox to ensure the correct behavior of this component.

Reply:
@vibdev Remove this line now as it is no longer being used